security/src/main/scala/pl/touk/nussknacker/ui/security/oauth2/BaseOAuth2Service.scala

```diff
@@ -23,11 +24,11 @@ class BaseOAuth2Service[
     } yield (authorizationData, userInfo)
   }
 
-  final def checkAuthorizationAndObtainUserinfo(accessToken: String): Future[(UserInfoData, Option[Deadline])] =
+  final def checkAuthorizationAndObtainUserinfo(accessToken: String): Future[(UserInfoData, Option[Instant])] =
     for {
-      deadline <- introspectAccessToken(accessToken)
+      expiration <- introspectAccessToken(accessToken)
```

Comment:
Can it be called `expirationTime`? Taka a look that we have also `expirationPeriod` in some places